sovereign-forge/src/phases.rs

```diff
@@ -89,7 +89,7 @@ pub trait PhasesModule:
 
         let esdt_safe_address = self.get_contract_address(&caller, ScArray::ESDTSafe);
 
-        self.deploy_fee_market(&esdt_safe_address, fee);
+        self.deploy_fee_market(&caller, &esdt_safe_address, fee);
```

Comment:
Please send caller as argument also to `deploy_header_verifier` and `deploy_chain_config` functions